src/backend/partaj/core/migrations/0045_referral_refuse_explanation.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 3.0.5 on 2021-07-30 07:31
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
We should remove unwanted migrations instead of removing them by adding a migration that does the opposite.